ui/src/db-tables/indexdb.ts

```diff
@@ -24,13 +24,13 @@ class ManagerDB extends Dexie {
 
   constructor() {
     super(WORKSPACE_INDEXDB_NAME);
-    this.version(1)
+    this.version(2)
```

Comment:
how indexdb works is that, you cannot delete prior db version, which sucks, you need to keep version(1) definition still

Reply:
done